src/rgw/const.py

```diff
@@ -17,6 +17,10 @@
 from enum import Enum
 
 COMPONENT_NAME = 'rgw'
+COMPONENT_SVC_NAME = 'rgw_s3'
+# service name be passed as --service parameter
+# e.g. rgw_setup --service={rgw_s3|all}
+SUPPORTED_SERVICE_NAMES = [COMPONENT_SVC_NAME, 'all']
```

Comment:
Remove this from this place. This variable can be part of individual code.

Reply:
done